### PR TITLE
testing please ignore

### PR DESCRIPTION
**Pull request by yardenshoham**
just testing LGTM count

### CODE REVIEW

**Review by techknowlogick — DISMISSED** [review]

**Review by delvh — APPROVED** [review]

**Review by delvh — CHANGES_REQUESTED** [review]
I can't allow this

File: .air.toml [hunk 1]
```diff
@@ -10,3 +10,5 @@ include_file = ["main.go"]
 include_dir = ["cmd", "models", "modules", "options", "routers", "services"]
 exclude_dir = ["modules/git/tests", "services/gitdiff/testdata", "modules/avatar/testdata", "models/fixtures", "models/migrations/fixtures", "modules/migration/file_format_testdata", "modules/avatar/identicon/testdata"]
 exclude_regex = ["_test.go$", "_gen.go$"]
+
```
Comment (techknowlogick) [inline .air.toml:13]:
```suggestion
```

**Review by techknowlogick — CHANGES_REQUESTED** [review]

**Review by techknowlogick — DISMISSED** [review]

**Review by techknowlogick — APPROVED** [review]

**Review by techknowlogick — DISMISSED** [review]
changes required

**Review by silverwind — APPROVED** [review]

**Review by silverwind — DISMISSED** [review]
need changes

**Review by jolheiser — APPROVED** [review]

**Review by jolheiser — DISMISSED** [review]
❌

**Comment by yardenshoham** [comment]
Closing because maintainers are sending me mixed signals

**Comment by yardenshoham** [comment]
Gonna test some stuff here, I don't want to open another PR so feel free to unsubscribe

**Review by pat-s — APPROVED** [review]
42

**Review by delvh — DISMISSED** [review]
Still a hard no.

**Review by jolheiser — CHANGES_REQUESTED** [review]
no

**Review by jolheiser — DISMISSED** [review]
still no

**Review by jolheiser — CHANGES_REQUESTED** [review]
actually....no

**Review by delvh — DISMISSED** [review]
Perhaps?

**Review by jolheiser — APPROVED** [review]

**Review by jolheiser — CHANGES_REQUESTED** [review]
nah

**Comment by yardenshoham** [comment]
i'm so confused, hostile project i'm out

**Pull request closed without merging**